feat: Blog post that introduces Pachli after project acceptance

nikclayton opened the pull request.

nikclayton commented:
@cla-bot check

nikclayton commented:
Marked this as a draft so there's no chance of actual submission until at least the `date` metadata is updated to reflect the desired publishing date.

quintessence commented on content/en/blog/2024/2024-06-pachli-introduction.md:
```diff
@@ -0,0 +1,68 @@
+
+- Mastodon: [@pachli@mastodon.social](https://mastodon.social/@pachli)
+- GitHub discussions: https://github.com/pachli/pachli-android/discussions
+- E-mail: [team@pachli.app](mailto:team@pachli.app)
```
Some things to add (feel free to fix formatting):

#### Quick Nivenly FAQ

**What is the Nivenly Foundation?**

The Nivenly Foundation is a non-profit co-op organization that seeks to provide governance and support to Open Source Projects. Our belief is that more active collaboration will improve open source projects, and our goal is to provide spaces and supporting mechanisms that enable communities and maintainers to work together.

**Do I need to be a Nivenly member to contribute code or docs to this project?**

No, you do not need to be a member of Nivenly to contribute to this project.

**Do I need to be a Nivenly member to participate in governance decisions for this project?**

Yes, you do need to be a Nivenly member to participate in governance decisions.

nikclayton replied:
I understand how this would make sense on the blog post that'll go on https://pachli.app, but is this necessary on the Nivenly blog post? Won't readers already know that?

quintessence replied:
We repeat it as often as possible, because people always ask.

nikclayton replied:
Sure thing. I've added it at the bottom.

quintessence commented on content/en/blog/2024/2024-06-pachli-introduction.md:
```diff
@@ -0,0 +1,68 @@
+
+- Mastodon: [@pachli@mastodon.social](https://mastodon.social/@pachli)
+- GitHub discussions: https://github.com/pachli/pachli-android/discussions
+- E-mail: [team@pachli.app](mailto:team@pachli.app)
```
Also somewhere in here there should be some prose similarish to the following:

The Pachli project has \[a CLA\]\(LINK\). In order to contribute code or docs you will need to sign the CLA, which can be done by \[instructions\]\(LINK\).

The Pachli project uses the LICENSE license.

nikclayton replied:
Good idea, thanks, and done. I've also mentioned and linked to the code of conduct.

nikclayton commented:
I've marked this as "Ready for review". The date metadata at the top will still need to be changed to reflect the desired date for it to go live, but I think this addresses the review comments so far.

nikclayton commented:
I've bumped the post date metadata to today, in case you decide to post this today.
